stable/metricbeat/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+description: A Helm chart to collect Kubernetes logs with metricbeat
+icon: https://www.elastic.co/assets/blt47799dcdcf08438d/logo-elastic-beats-lt.svg
+name: metricbeat
+version: 0.3.0
+appVersion: 6.2.3
+home: https://www.elastic.co/products/beats/metricbeat
+sources:
+- https://www.elastic.co/guide/en/beats/metricbeat/current/index.html
+maintainers:
```

Comment:
add an OWNERS file too

Reply:
Sorry, but I'm not sure what OWNERS file is. 
Can you show me some example charts including OWNERS file?

Reply:
See https://github.com/helm/charts "Owning and maintaining a chart"

Reply:
I added OWNERS file. Thank you!